parkings/forecasts/cache_parking_counts.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+
+from parkings.models import ActiveParking, Parking, ParkingArea, Region
+
+
+def cache_parking_counts_by_region():
+    parkings_start_instance = (
+        ActiveParking.objects.filter(region__isnull=False, is_forecast=False)
+        .order_by("time")
+        .last()
+    )
+    if parkings_start_instance:
+        parkings_start = parkings_start_instance.time
+    else:
+        parkings_start = Parking.objects.all().order_by("time_start").first().time_start
+
+    parkings_end = Parking.objects.all().order_by("-time_start").first().time_end
+
+    current_datetime = datetime.datetime(
+        parkings_start.year,
+        parkings_start.month,
+        parkings_start.day,
+        parkings_start.hour,
+        tzinfo=parkings_start.tzinfo,
+    )
```

Comment:
Use `current_datetime = parkings_start.replace(minutes=0, seconds=0, microseconds=0)` here since this could break when there is a DST change. And it's not really "current" datetime, rather `start_hour` or something like that.

Reply:
Actually, after checking the code below more carefully, even better variable name would be just `hour`, since it's going to be iterating over all the hours.